utils/line-directive

```diff
@@ -61,7 +61,7 @@ def fline_map(target_filename):
 def map_line_to_source_file(target_filename, target_line_num):
     """
     >>> from tempfile import *
-    >>> t = NamedTemporaryFile()
+    >>> t = NamedTemporaryFile(delete=False)
```

Comment:
@hughbe When you do something for esoteric (i.e. platform-specific, optimization, or otherwise subtle) reasons like this, you should leave a comment explaining why it's there.  Otherwise I or someone else is liable to think ”oh, that's dumb, I'll just clean it up” and put things back the way they were.  I don't understand why  this is needed on Windows, but perhaps if it is, we should encapsulate it in our own wrapper for `NamedTemporaryFile`.